.env.sample

```diff
@@ -1,3 +1,5 @@
 PLANS_TABLE_NAME=
+NEXT_PUBLIC_URL=
 NEXT_PUBLIC_API_URL=
+NEXT_PUBLIC_SMS_API_URL=
```

Comment:
Does this need to be public?
If it's only used in the backend we can remove the `NEXT_PUBLIC` prefix.

Reply:
Hm, yeah, doesn't seem like it has to be